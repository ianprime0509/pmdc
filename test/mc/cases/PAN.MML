; Output: PAN.M
; Console: PAN.OUT

#Title Pan

A c p1c p2c p3c px-3c px-1c px0c px+2c px+4c px0,1c
