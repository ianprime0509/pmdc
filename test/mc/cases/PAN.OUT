 .MML file --> .M file Compiler ver 4.8s
		Programmed by M.Kajihara(KAJA) 2023/09/23

Compile Completed.
